dictionary_app/app.py

```diff
@@ -0,0 +1,87 @@
+import pymongo
+from flask import Flask, render_template, request, redirect
+import requests
+
+client = pymongo.MongoClient("mongodb+srv://base_user:base_user_password@cluster0.dbcb9.mongodb.net/first")
+db = client.first
+collection = db.gokberk
```

Comment:
I suggest you to change the name of the "collection" variable to avoid conflicts with other example projects

Reply:
> I suggest you to change the name of the "collection" variable to avoid conflicts with other example projects

I agree. You've made good points. After these changes, this app will be OK